lib/hipchat/api_version.rb

```diff
@@ -302,6 +302,15 @@ def view_config
         }[version]
       end
 
+      def user_room_config
+        {
+          'v2' => {
```

Comment:
Can you raise an InvalidApiVersion exception when the api is set to v1 (see history_config for an example)